Dockerfile_webhook

```diff
@@ -0,0 +1,35 @@
+# https://hub.docker.com/_/debian
```

Comment:
The webhook should just be included with the site, there's no reason to launch it separately.

Reply:
That'll mean doing `podman run --entrypoint=janitor-webhook ghcr.io/jelmer/janitor/site:latest`